platform/frontend_connectors/schema_transformer.go

```diff
@@ -1185,3 +1186,26 @@ func (s *SchemaCheckPass) applyMatchOperator(indexSchema schema.Schema, query *m
 	return query, nil
 
 }
+
+func (s *SchemaCheckPass) applyClusterFunction(currentSchema schema.Schema, query *model.Query) (*model.Query, error) {
```

Comment:
```suggestion
// applyFromClusterExpression transforms query so that `FROM table` becomes `FROM cluster(clusterName,table)` if applicable
func (s *SchemaCheckPass) applyFromClusterExpression(currentSchema schema.Schema, query *model.Query) (*model.Query, error) {
```